src/modules/chwd/main.py

```diff
@@ -75,7 +75,7 @@ def run():
                 "exist, doing nothing".format(root_mount_point))
 
     # run the command in chroot
-    shell_command = ["arch-chroot", root_mount_point, "/etc/calamares/scripts/install-gpu-drivers"]
+    shell_command = ["arch-chroot", root_mount_point, "chwd --autoconfigure"]
```

Comment:
```suggestion
    shell_command = ["arch-chroot", root_mount_point, "chwd", "--autoconfigure"]
```

arch-chroot pass arguments as-is